profile/src/main/resources/profileschema/0.2/datahelix.schema.json

```
@@ -0,0 +1,557 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://scottlogic.com/datahelix.json",
+  "title": "The Scott Logic DataHelix Profile Schema",
+  "type": "object",
+  "additionalProperties": false,
+  "required": ["schemaVersion", "fields", "rules"],
+  "properties": {
+    "additionalProperties": false,
+    "schemaVersion": {
+      "$ref": "#/definitions/schemaVersion"
+    },
+    "description": {
+      "title": "A description of what data the profile is modelling",
+      "type": "string"
+    },
+    "fields": {
+      "title": "Defines the fields that data will be produced for. field names must begin with an alphabetic character.",
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "additionalItems": false,
+      "items": {
+        "$ref": "#/definitions/fieldName"
+      }
+    },
+    "rules": {
+      "title": "The Rules defining the data to be output",
+      "type": "array",
+      "additionalItems": false,
+      "items": {
+        "title": "A set of Rule objects.",
+        "type": "object",
+        "additionalProperties": false,
+        "required": ["constraints"],
+        "properties": {
+          "rule": {
+            "title": "A collection of constraints. Test case generation revolves around rules, in that the generator will output a separate dataset for each rule, wherein each row violates the rule in a different way.",
+            "type": "string"
+          },
+          "constraints": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/constraint"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "schemaVersion": {
+      "title": "The version of the DataHelix profile schema",
+      "enum": ["0.1", "0.2"]
```

Following our discussion this morning, this should be 
```suggestion
      "const": "0.2"
```